src/apps/relay/userdb.c

```diff
@@ -685,6 +692,15 @@ int check_new_allocation_quota(uint8_t *user, int oauth, uint8_t *realm) {
     free(username);
     ur_string_map_unlock(rp->status.alloc_counters);
   }
+
+#ifndef _MSC_VER
+  global_allocation_count++;
+  TURN_LOG_FUNC(TURN_LOG_LEVEL_INFO, "Global turn allocation count incremented, now %ld\n", global_allocation_count);
```

Comment:
This should not be INFO
DEBUG seems like a better fit - would we need a new log line for every allocation? We already have a log line for every allocation

Reply:
I agree, I will make that change.